app/src/main/java/org/oppia/android/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -49,6 +54,11 @@ class ExplorationActivityPresenter @Inject constructor(
   private lateinit var context: Context
   private var backflowScreen: Int? = null
 
+  private var isCheckpointingEnabled: Boolean = false
+
+  private lateinit var oldestExplorationId: String
```

Comment:
I think this was named considering the `oldest progress` concept in mind but here we can use `explorationId` as it will always be either null or Id of the current exploration.

WDYT?

Same for Title.

Reply:
@rt4914 This will not be the id of the current exploration instead it is the explorationId of the exploration which has the oldest save progress in the database.

Reply:
renamed this to `oldestCheckpointExplorationId`

Reply:
Ack.